pkg/errors/errors.go

```diff
@@ -1,31 +1,37 @@
-package linodego
+package errors
 
 import (
 	"fmt"
-	"log"
 	"net/http"
 	"strings"
 
 	"github.com/go-resty/resty/v2"
 )
 
 const (
-	// ErrorFromString is the Code identifying Errors created by string types
-	ErrorFromString = 1
-	// ErrorFromError is the Code identifying Errors created by error types
-	ErrorFromError = 2
-	// ErrorFromStringer is the Code identifying Errors created by fmt.Stringer types
-	ErrorFromStringer = 3
+	// FromString is the Code identifying Errors created by string types
+	FromString = iota + 1
+	// FromError is the Code identifying Errors created by error types
+	FromError
+	// FromStringer is the Code identifying Errors created by fmt.Stringer types
+	FromStringer
 )
 
-// Error wraps the LinodeGo error with the relevant http.Response
+// Error represents an Error in the context of an API method call.
 type Error struct {
 	Response *http.Response
 	Code     int
 	Message  string
 }
 
-// APIErrorReason is an individual invalid request message returned by the Linode API
+func (e Error) Error() string {
+	return fmt.Sprintf("[%03d] %s", e.Code, e.Message)
+}
+
+// Error implements error.
+var _ error = (*Error)(nil)
```

Comment:
Is this like a compile time assertion/test?

Reply:
Oops yeah, I dunno why I left this in-- I'll remove it.

But, FWIW, https://golang.org/doc/effective_go.html#blank_implements.

Reply:
I thought it was fine, was just curious.